pages/user guides/code_research.md

```diff
@@ -0,0 +1,51 @@
+---
+toc: false
+title: Tidal Code Research Program
+keywords: code, analyze, source code
+last_updated: Novemvber, 2023
+summary: "Participate in the Code Research Program and gain even more insights from your code"
+sidebar: main_sidebar
+folder: userguides
```

Comment:
I am not sure I see this in our sidebar/navigation system is that purposeful?

Reply:
Yea it's not. I was thinking to have the option to release it before the feature is available. I will open a PR to add it.